AgentAssistAngular/src/app/components/assist/assist.component.ts

```diff
@@ -497,6 +500,7 @@ export class AssistComponent implements OnInit, OnDestroy {
   }
 
   handlePopupEvent(popupObject) {
+    this.conversationSummaryObj = popupObject;
```

Comment:
assign conversationSummaryObj only for summary event and on close.

Reply:
> assign conversationSummaryObj only for summary event and on close.

Currently, we are handling the conversationSummaryObj on Submit and onClose btn click.